usrsctplib/user_recv_thread.c

```diff
@@ -721,7 +724,7 @@ recv_function_udp(void *arg)
 		msg.msg_controllen = sizeof(cmsgbuf);
 		msg.msg_flags = 0;
 
-		ncounter = n = recvmsg(SCTP_BASE_VAR(userspace_udpsctp), &msg, 0);
+		ncounter = n = (int)recvmsg(SCTP_BASE_VAR(userspace_udpsctp), &msg, 0);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/a8adc4ae7cdfd014637ae3c0caf55d2bfda5ffc5